### PR TITLE
fix: rename npm build script to prepare

### PR DESCRIPTION
**Pull request by remcohaszing**
This allows to install karma directly from git.

### CODE REVIEW

**Comment by remcohaszing** [comment]
To test this, simply run:

```
npm install 'github:remcohaszing/karma#prepare'
```

**Comment by johnjbarton** [comment]
Since you removed the 'build' item, several other places need to be fixed with build -> prepare:
https://github.com/karma-runner/karma/search?utf8=%E2%9C%93&q=npm+run+build&type=

File: package.json [hunk 1]
```diff
@@ -455,8 +455,8 @@
     "unlink": "node --eval \"require('fs').unlinkSync(require('path').resolve(__dirname, 'node_modules', 'karma'))\"",
     "init": "rm -rf node_modules/karma && cd node_modules && ln -nsf ../ karma && cd ../",
     "init:windows": "(IF EXIST node_modules\\karma (rmdir node_modules\\karma /S /q)) && npm run link",
-    "appveyor": "npm run build && npm run test:appveyor",
```
[the remark this answers is not in the document]
Comment (lusarz) [reply package.json]: I see it's changed into:

    "appveyor": "npm run test:appveyor"

so "npm run build" part was removed. I guess `prepare` script runs automatically, so it's ok.

**Comment by lusarz** [comment]
I've tried to restart travis build a few times, but it fails with that message:
error

I have no idea why :( @remcohaszing would you mind to rebase your branch on current master ? Maybe it would help...

**Comment by johnjbarton** [comment]
Sorry this did not work out. I'm going to close a lot of older pull requests as the source code has changed a lot over the last year. 
Thanks!

**Pull request closed without merging**